client/src/modules/Mentor/pages/Interviews/components/StudentInterview.tsx

```diff
@@ -1,40 +1,44 @@
 import { Col, Row, Button, Typography, Space, message, Popconfirm } from 'antd';
 import { GithubAvatar } from 'components/GithubAvatar';
 import GithubFilled from '@ant-design/icons/GithubFilled';
-import { DecisionTag, getInterviewFeedbackUrl } from 'domain/interview';
+import { DecisionTag, getInterviewFeedbackUrl, InterviewStatus } from 'domain/interview';
 import { CourseService, MentorInterview } from 'services/course';
 import css from 'styled-jsx/css';
 import { useState } from 'react';
 import { CloseCircleOutlined } from '@ant-design/icons';
+import { TaskDtoTypeEnum } from 'api';
 
 export function StudentInterview(props: {
   interview: MentorInterview;
+  interviewTaskType: TaskDtoTypeEnum;
   template?: string | null;
   courseAlias: string;
   courseId: number;
 }) {
-  const { interview, template, courseAlias, courseId } = props;
+  const { interview, interviewTaskType, template, courseAlias, courseId } = props;
   const { student, completed } = interview;
 
-  const [interviewFeedbackUrl, setInterviewFeedbackUrl] = useState('');
   const [isInterviewCompleted, setInterviewCompleted] = useState(completed);
   const [popconfirmOpen, setPopconfirmOpen] = useState(false);
+
   const courseService = new CourseService(courseId);
+  const isCoreJsInterview = interviewTaskType === TaskDtoTypeEnum.Interview;
+  const interviewStatus = !isInterviewCompleted ? interview.status : InterviewStatus.Completed;
```

Comment:
Let's avoid excessive negation operation
```suggestion
  const interviewStatus = isInterviewCompleted ? InterviewStatus.Completed : interview.status;
```